src/ExportSqlDialog.ui

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>ExportSqlDialog</class>
+ <widget class="QDialog" name="ExportSqlDialog">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>497</width>
+    <height>300</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Export SQL...</string>
```

Comment:
This seems like a hard coded English string.  Can we make this so it is translated?

We have French, Russian, Chinese, and other translation string files already, but I am not sure how they work with .ui files. :wink: